Update _index.md (#22976)
DataDog/documentation: content/en/synthetics/private_locations/_index.md

[102]: https://docs.docker.com/engine/install/
[103]: https://www.datadoghq.com/legal/eula/

{{% /tab %}}
{{% tab "Helm" %}}

> Private locations are Kubernetes deployments that you can install on your Kubernetes cluster with Helm. You can access the [helm chart][101] on Github. It can run on Linux-based Kubernetes.**\***

Reviewer: ```suggestion
Private locations are Kubernetes deployments that you can install on your Kubernetes cluster with Helm. The [helm chart][101] can run on Linux-based Kubernetes.
```

> **Note**: Use and operation of this software is governed by the [End-User License Agreement][102].

Reviewer: ```suggestion
**Note**: Use and operation of this software is governed by the [End-User License Agreement][103].
```

> **\*** **Use and operation of this software is governed by the End User License Agreement available [here][102].**

Reviewer: ```suggestion
**Note**: Use and operation of this software is governed by the [End-User License Agreement][102].
```
